face_detection_masking/test_face_detection_masking.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+import numpy as np
+import cv2
+
+from main import face_detection, process_img, create_mask
+from exceptions import InvalidFaceImage
+
+
+def test_smiling_face():
+    uri = (
+        "https://image.freepik.com/free-photo/"
```

Comment:
if we're checking several images, we might as well download and check in this image too. URLs are subject to change and will make this test obsolete in the future. Another approach is to store these images in a test bucket then have our own urls so that they will not go away

Reply:
We need to have our own URLs since the face_dectection() needs a URL as an argument.

Reply:
I am having a hard time accessing the test bucket via local dev env. I will handle this in a separate ticket.

Reply:
Yeah, I guess you can create a test bucket and upload your image there. I think this way it makes sure you can always have a valid URL.
BTW, why is the var named uri? Is this a typo?

Reply:
No, it's not. Since we work with images stored in the bucket we reference it directly using a URI in form of `gs://*`.

Reply:
sounds good! Let's do this in a different PR then (could you make a Jira task for this?). You can maybe have a test buckets, put these images in there then get downloadable urls for them but make sure to restrict access so only those with proper access can run these tests.